src/client/customers.rs

```diff
@@ -235,6 +235,203 @@ pub struct Address {
+    /// method.
+    pub auto_collection: bool,
+    /// The difference between the invoice date and the issue date for the invoice. If due on issue,
+    /// set this to `0`.
+    pub net_terms: u64,
+    /// An optional memo to display on the invoice
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub memo: Option<&'a str>,
+}
+
+/// The parameters used to create a customer credit ledger entry.
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize)]
+pub struct AddIncrementCreditLedgerEntryRequestParams<'a> {
+    /// The amount to credit the customer for.
+    pub amount: serde_json::Number,
+    /// An optional description for the credit operation.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub description: Option<&'a str>,
+    /// The date on which the block's balance will expire.
+    #[serde(with = "time::serde::rfc3339::option")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub expiry_date: Option<OffsetDateTime>,
+    /// The date on which the block's balance will become available for use.
+    #[serde(with = "time::serde::rfc3339::option")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub effective_date: Option<OffsetDateTime>,
+    /// The price per credit.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub per_unit_cost_basis: Option<&'a str>,
+    /// Invoicing settings for the credit increment request.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub invoice_settings: Option<CreditLedgerInvoiceSettingsRequestParams<'a>>,
+}
+
+/// The reason for a void operation.
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Deserialize_enum_str, Serialize_enum_str)]
+pub enum VoidReason {
+    /// The credits are being returned to the originator.
+    #[serde(rename = "refund")]
+    Refund,
+}
+
+/// The parameters used to void a customer credit ledger entry.
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize)]
+pub struct AddVoidCreditLedgerEntryRequestParams<'a> {
+    /// The number of credits to void.
+    pub amount: serde_json::Number,
+    /// The ID of the credit ledger block to void.
+    pub block_id: &'a str,
+    /// An optional reason for the void.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub void_reason: Option<VoidReason>,
+    /// An optional description for the void operation.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub description: Option<&'a str>,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Deserialize, Serialize)]
+pub struct CustomerCreditBlock {
+    /// The Orb-assigned unique identifier for the credit block.
+    pub id: String,
+    /// The remaining credit balance for the block.
+    pub balance: serde_json::Number,
+    /// The date on which the block's balance will expire.
+    #[serde(with = "time::serde::rfc3339::option")]
+    pub expiry_date: Option<OffsetDateTime>,
+    /// The price per credit.
+    pub per_unit_cost_basis: Option<String>,
```

Comment:
you don't want skip if none here?

Reply:
This struct is only used to deserialize an API response, so it isn't necessary IMO.